mozilla/foundation.mozilla.org
# updated "url" to "embed url" in video block

Closes #8328
Related PRs/issues #

**Link to sample test page**:
[Link to blog page using video block](https://foundation-s-video-bloc-fxwejj.mofostaging.net/en/blog/initial-test-blog-post-with-fixed-title/)
[Link to edit page in CMS to view new "embed URL" title ](https://foundation-s-video-bloc-fxwejj.mofostaging.net/cms/pages/13/edit/)(Credentials: review/review)


Hi @kristinashu!

Thank you for taking the time to review this PR 😄 

After looking through the original ticket, my first thought for a solution was to update the help text, however, after taking a look, it seems that we do a pretty thorough job explaining how to get the correct link for the video block.

This leads me to believe the next best thing is what you suggested, which was to update the field "url" to "embed URL".

Another idea, though it may be outside the scope of this ticket, would be "how to" videos for certain streamfield blocks. I would be happy to make these if the team decides this is something we should move forward with! 

**Steps to test**:

1. If you would like to verify that the video block still works, please visit the first link above.
2. If you would like to take a look at the new title for the field, please visit the second link above and log in using the credentials (review/review), though here is a screenshot below:
<img width="1003" alt="image" src="https://user-images.githubusercontent.com/18314510/163497446-fe2df178-d72a-42fe-aaa7-597d8e746c10.png">
3. If everything is looking as expected, testing is complete!
